Doc/library/profile.rst

```diff
@@ -299,6 +299,15 @@ functions:
       Create a :class:`~pstats.Stats` object based on the current
       profile and print the results to stdout.
 
+      The *sort* parameter specifies the sorting order of the displayed
+      statistics. It accepts a single key or a tuple of keys to enable
+      multi-level sorting. A key can be either an integer, a string, or a
+      SortKey enum identifying the basis of a sort (for example: ``-1``,
+      ``'time'``, ``'name'``, ``SortKey.TIME``, or ``SortKey.NAME``).
```

Comment:
```suggestion
      multi-level sorting. When passing a tuple of keys, each key must be
      either a string or a SortKey enum, excluding integers. However,
      if only a single key is provided, it can be an integer, a string, or a
      SortKey enum. These keys serve to identify the basis of the sort,
      such as ``'time'``, ``'name'``, ``SortKey.TIME``, or ``SortKey.NAME``,
      among others."
```